app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str | None:
+    cmd = command.split()
+    if cmd[0] != "mv":
+        return f"{cmd[0]} not a move command"
+
+    path = cmd[2].split("/")
+    new_file = path.pop()
+    path = "/".join(path)
+
+    if not os.path.isdir(path):
+        os.makedirs(path)
+    try:
+        with open(cmd[1], "r") as source, \
```

Comment:
I think it's better to use parentheses here.
```python
with (open...,
      open...):

Reply:
I didn't understand what you mean here. I use "\" just for splitting

Reply:
Aaa understand! Thank you! Give me 5 minutes to change. It's cool!

Reply:
Oops, for some reason your reply didn't load when I opened the page last time. Good thing you understood yourself! :)

Reply:
It seemed that you were proposing to shorten the code due to fewer brackets, but I tried it and realized that it's not - please approve ;) In general, I probably need to be more sure in myself. Thank you!

Reply:
Yes, this would not shorten the code. It would add one more symbol to it.
However, the mentors at Mate seem to be much more fond of using parentheses than backslashes for line breaks. It also infected me.